api/HTMLAllCollection.json

```diff
@@ -14,10 +14,10 @@
             "version_added": "≤18"
           },
           "firefox": {
-            "version_added": true
+            "version_added": "32"
```

Comment:
The `document.all` entry says Firefox 24. What kind of a thing was `document.all` in Firefox 24 if not an `HTMLAllCollection`?

Reply:
Seems that it was an `HTMLCollection` rather than an `HTMLAllCollection`.  This appears to be common amongst all of the browsers.

Reply:
Aha, this situation again. Well that explains it!